liferay-echo/liferay-portal
# LPS-166852 BasicDocumentSingleFormVariationInfoCollectionProvider not available in the UI after enabling it

## Motivation

[Link to bug](https://issues.liferay.com/browse/LPS-LPS-166852)

There's a bug that won't show collection providers registered to FileEntry class, just for DLFileEntry class

The proposed solution would be to revert the changes in cf81bd2d7da2980515defb3f52c339252a7ef35d for the specific method, since we add the className to be taken into account


## Change summary:

* Partially revert the changes introduced in cf81bd2d7da2980515defb3f52c339252a7ef35d
* Add unit tests so that this won't fail again in the future


## How to test the changes

* Go to pogo shell in applications menu
* Enable component: scr:enable com.liferay.document.library.web.internal.info.collection.provider.BasicDocumentSingleFormVariationInfoCollectionProvider
* Go to any site
* Add a content page
* Add a collection fragment to the page
* Click on it to configure
* Click on Collection Providers Tab
* The Basic Documents provider should be shown
